gradle-baseline-java/src/test/groovy/com/palantir/baseline/plugins/BaselineReproducibilityIntegrationSpec.groovy

```diff
@@ -109,4 +109,33 @@ class BaselineReproducibilityIntegrationSpec extends IntegrationSpec {
         def output = runTasksSuccessfully("check")
         output.getStandardOutput().contains("> Task :checkExplicitSourceCompatibility SKIPPED")
     }
+
+    def 'task passes when toolchains are used'() {
```

Comment:
I couldn't get this to run locally (verified by inverting checks in existing tests and rerunning). I need to figure out what's going on.

Reply:
Fixed in #1916